gbpservice/nfp/orchestrator/db/nfp_db_NSD.py

```diff
@@ -0,0 +1,130 @@
+from gbpservice.nfp.orchestrator.db.nfp_db import NFPDbBase
+
+from gbpservice.nfp.orchestrator.db import common_db_mixin
+from gbpservice.nfp.orchestrator.db import nfp_db_model
+
+from gbpservice.nfp.core import log as nfp_logging
+LOG = nfp_logging.getLogger(__name__)
+
+
+class NFPDbBaseNSD(NFPDbBase):
+    def __init__(self, *args, **kwargs):
+        super(NFPDbBaseNSD, self).__init__(*args, **kwargs)
+
+    def _set_plugged_in_port_for_nfd_interface(self, session, nfd_interface_db,
+                                               interface, is_update=False):
+        plugged_in_port_id = interface.get('plugged_in_port_id')
+        if not plugged_in_port_id:
+            if not is_update:
+                nfd_interface_db.plugged_in_port_id = None
```

Comment:
Lets discuss on how to avoid the duplicate code. Looks like we have two files with enterprise stuff in enterprise file.